dump/mapping/prepbufr_acft_profiles.yaml

```diff
@@ -0,0 +1,247 @@
+# (C) Copyright 2023 NOAA/NWS/NCEP/EMC
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+
+bufr:
+  subsets:
+    - AIRCFT
+    - AIRCAR
+
+  group_by_variable: prepbufrDataLevelCategory
+  variables:
+    # ObsType
+    airTemperatureObservationType:
+      query: "*/TYP"
+    specificHumidityObservationType:
+      query: "*/TYP"
+    windObservationType:
+      query: "*/TYP"
+
+    # MetaData
+    timestamp:
+      timeoffset:
+        timeOffset: "*/DHR"
+        transforms:
+          - scale: 3600
+        referenceTime: "1970-01-01T00:00:00Z"
+    prepbufrDataLevelCategory:
+      query: "*/PRSLEVLA/CAT"
+    stationIdentification:
+      query: "*/SID"
+    aircraftFlightNumber:
+      query: "[*/ACID, */ACID_SEQ/ACID]"
+    longitude:
+      query: "*/PRSLEVLA/DRFTINFO/XDR"
+      transforms:
+        - wrap: [ -180.0, 180.0 ]
+    latitude:
+      query: "*/PRSLEVLA/DRFTINFO/YDR"
+    obsTimeMinusCycleTime:
+      query: "*/PRSLEVLA/DRFTINFO/HRDR"
+    heightOfStationMetaData: #Formerly aircraftFlightLevel
+      query: "*/PRSLEVLA/Z___INFO/Z__EVENT{1}/ZOB"
+    stationElevationMetaData:
+      query: "*/ELV"
+      type: float
+    pressureMetaData:
+      query: "*/PRSLEVLA/P___INFO/P__EVENT{1}/POB"
+      transforms:
+        - scale: 100
+    temperatureEventCode:
+      query: "*/PRSLEVLA/T___INFO/T__EVENT{1}/TPC"
+    instantaneousAltitudeRate:
+      query: "*/PRSLEVLA/IALR"
+    aircraftFlightPhase:
+      query: "*/PRSLEVLA/ACFT_SEQ{1}/POAF"
+
+    #ObsValue
+    airTemperatureObsValue:
+      query: "*/PRSLEVLA/T___INFO/T__EVENT{1}/TOB"
+      transforms:
+        - offset: 273.15
+    specificHumidityObsValue:
+      query: "*/PRSLEVLA/Q___INFO/Q__EVENT{1}/QOB"
+      type: float
+      transforms:
+        - scale: 0.000001
+    windEastwardObsValue:
+      query: "*/PRSLEVLA/W___INFO/W__EVENT{1}/UOB"
+    windNorthwardObsValue:
+      query: "*/PRSLEVLA/W___INFO/W__EVENT{1}/VOB"
+
+    #QualityMarker
+    pressureQualityMarker:
+      query: "*/PRSLEVLA/P___INFO/P__EVENT{1}/PQM"
+    airTemperatureQualityMarker:
+      query: "*/PRSLEVLA/T___INFO/T__EVENT{1}/TQM"
+    specificHumidityQualityMarker:
+      query: "*/PRSLEVLA/Q___INFO/Q__EVENT{1}/QQM"
+    windEastwardQualityMarker:
+      query: "*/PRSLEVLA/W___INFO/W__EVENT{1}/WQM"
+    windNorthwardQualityMarker:
+      query: "*/PRSLEVLA/W___INFO/W__EVENT{1}/WQM"
+
+    #ObsError
+    airTemperatureObsError:
+      query: "*/PRSLEVLA/T___INFO/T__BACKG/TOE"
+    specificHumidityObsError:
+      query: "*/PRSLEVLA/Q___INFO/Q__BACKG/QOE"
+      type: float
+      transforms:
+        - scale: 0.1
+    windObsError:
+      query: "*/PRSLEVLA/W___INFO/W__BACKG/WOE"
+
+
+encoder:
+  globals:
```

Comment:
`globals` needs update.

IODA convention changed its requirement for `globals`
You can find the valid `global` in [ObsSpae.yaml](https://github.com/JCSDA-internal/ioda/blob/feature/ObsSpace_Validator/share/ioda/yaml/validation/ObsSpace.yaml) in ioda. This is our working branch to add the elements for new variables.  

Here is the example:
https://github.com/NOAA-EMC/spoc/blob/b7428ac1c602314633257fa9f1763e8d389f12d0/dump/mapping/bufr_scatwnd_ascat.yaml#L55-L90

Reply:
Fixed. Thanks